### DISCUSSION TITLE
关于项目未来的发展方向

### DISCUSSION THREAD

**Discussion by Poritani-Makoto** [kono-dada/Sakuranotoki-Chinese#25] [General]
在b站看到了一系列名为“樱之刻剧情流程”的专栏，其中对非常多的对话已经自行进行了翻译了。有点想征求一下译者的意见，看看能不能放进汉化里来（在署名的前提下），不过这么思考的时候想到了两个问题：
1. 译者不一定会同意，哈哈
2. 这件事其实让我想问一下kono-dada老师的初衷或者关于未来发展方向的期望，即：
樱之刻gpt4翻译版，其最核心的目的是什么？如果只是为了给玩家提供一个良好的翻译版本，那自然这么做也没有问题；但还有一种可能或许是“探索现阶段技术下gpt4协助翻译的极限”？目前为止我做的所有改动都是基于gpt4翻译的，只是参考了原文之后进行一定程度的调整，虽然可能得到的结果类似，但是操作模式在本质上与“人力看完原文后直接翻译成中文，未经过任何机器辅助”有根本的区别。如果本项目实质上是一场实验，探究半吊子日语玩家（或日语大佬）**基于机翻进行调整（而不是从源文件直接人力翻译）**的效率与质量极限，由此作为一种实验去探索全新的翻译流程，并优化流程。如果是这个思路，那可能加入人工翻译的想法也就不合适了。如此一来虽然得到的效果可能不如掺杂了人工翻译的版本，但可能潜在地能够对未来gpt4翻译工作有更大的帮助？
其实这两种思路我都完全可以理解，不过还是想咨询一下kono老师的想法

**Comment by kono-dada** [comment]
其实GPT翻译的目标就是迅速提供一个95分的翻译，即追求速度和观感，而牺牲小部分准确程度和文学性。传统机器翻译虽然速度快，但句子缺乏逻辑性，语句混乱，人称残缺，有时会出现过于离谱费解的句子。GPT4翻译的结果则顺应汉语语言习惯，甚至不需要润色，使阅读的流畅感大幅提升，也就给予玩家一个良好的游戏体验。

然而GPT4的翻译结果依旧存在问题，归结起来分为以下几点。有些问题可以通过开源社区召集大量普通玩家来贡献修订，有些问题必须依赖专业汉化组：
1. 性别可能会搞混，省略的主语可能错误。这一点普通玩家简单地提交问题就可以修正。
2. 文学性、哲学意味强的剧情高潮段落。这需要专业的汉化组，但一个游戏里的高光时刻并不多，专业的汉化组也许可以在一周内就能完成这些需要专业能力的汉化。
3. neta出处，名作引用等。如果可以激励见多识广的老玩家出手，这些也不是问题。
4. 专有名词的翻译问题。这本质上不是个问题，如果在翻译开工前就做好专有名词的提取，就可以规避这个问题。

总的来说，GPT4翻译的初衷是利用人多力量大的社区，在极短时间内（可能一到两周）共同完成一个游玩流畅度95分的作品。当然，这一切的前提是汉化项目的宣传足够到位，有激励玩家做出贡献的手段，并且得到高水平汉化者的认可。我们不希望后续的佳作重蹈樱之诗5年没有翻译的覆辙，而实际上甚至三个月以上的等待时间都是极为漫长的。

**Comment by kono-dada** [comment]
本次翻译是在5月底才开始的，已经过去两个月。再加上我也是第一次做这种事，考虑并不周全，所以并没有吸引到足够的人气来做贡献。

**Comment by Poritani-Makoto** [comment]
若真的希望在名作发售后立刻开始召集大量普通玩家和专业汉化组进行共同维护翻译，其实还有一点需要注意，也就是仍然需要比较传统的明确的分工，不然处理merge conflict将会非常蛋疼，这方面可能也会需要比较专业的校对来判断处理才能最大化效益

**Comment by kono-dada** [comment]
有一说一，或许工程量或许没有我想象的那么高。我感觉5个普通人加1个专业汉化就可以在1周内做出很高质量的翻译

**Comment by Poritani-Makoto** [comment]
确实，挑刺顺便改下其实比从头开始考虑要快太多了，下次准备拿什么作品再试试？变态监狱？据说那个现在也还没有汉化。或者在樱之响出以前先做一波宣传，挑战速通（有生之年
其实关于各种文学哲学的出处，只要具备一定使用搜索引擎的能力以及判断信息来源可靠性的能力，普通玩家也能够做到的。专业玩家应该主要还是注重于重要段落的语言艺术性并保证翻译准确性，如《诗》中的草地对话。

**Comment by Poritani-Makoto** [comment]
其实我是想问，你对这个项目（特指樱之刻gpt4汉化）未来是否有进一步的想法呢，只是好奇，毕竟大家不是上学就是上班，这东西也就业余搞一搞（
可能的几个方向有
1. 继续放在这，等有缘人随缘优化完善
2. 主动收集各种b站上公开的汉化询问是否能够加入对应译文到库里
3. 进行相关总结讨论，结合这次尝试的实际结果，讨论gpt4汉化的可行性、流程与优劣，让更多的人知道并参与到这一模式中来

**Comment by kono-dada** [comment]
1和3吧。我个人觉得没有必要花费大量精力打造一个完美的汉化，毕竟汉化组们正在为此努力。再者，现在《刻》的潜在玩家要么已经玩了机翻要么就硬等汉化组，我们或许没法吸引到足够的人来玩此版翻译。所以说，我并不会去主动地花时间在这里

**Comment by kono-dada** [comment]
不知道，我不算galgame的重度玩家，所以也并没有特别热衷翻译。有机会或者兴趣来了再试。反正我也开源了我的翻译脚本。
